### PR TITLE
Add features section to the home page

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
Add features section to the index page

### CODE REVIEW

File: exampleSite/content/_index/items-1/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
++++
+fragment = "items"
+weight = 100
+background = "light"
+
+title = "Features"
+subtitle= "Syna Comes with a lot of features"
```
Comment (stp-ip) [inline exampleSite/content/_index/items-1/index.md]: Comes -> comes

File: exampleSite/content/_index/items-1/configurable.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
++++
+title = "Configurable"
+weight = 20
+icon = "fas fa-cubes"
++++
+
+Everything in a website using Syna theme from navigation links to different sections of each page is configurable
```
Comment (stp-ip) [inline exampleSite/content/_index/items-1/configurable.md]: High flexibility to easily configure navigation, sections or whole pages

File: exampleSite/content/_index/items-2/customizble.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
++++
```
Comment (stp-ip) [inline exampleSite/content/_index/items-2/customizble.md]: typo in filename

File: exampleSite/content/_index/items-2/extendible.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
++++
```
Comment (stp-ip) [inline exampleSite/content/_index/items-2/extendible.md]: extendible -> extensible?

File: exampleSite/content/_index/items-2/customizble.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+title = "Customizable"
+weight = 10
+icon = "fas fa-pencil-alt"
++++
+
+We use Bootstrap which allows for cutomizing every aspect of the style using variables
```
Comment (stp-ip) [inline exampleSite/content/_index/items-2/customizble.md]: Is that feature really exposed for a normal user? Might actually skip this feature.

File: exampleSite/content/_index/items-1/react.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
++++
```
Comment (stp-ip) [inline exampleSite/content/_index/items-1/react.md]: Would make react support an "item" fragment instead. Big enough feature and the others could be reduced.

File: exampleSite/content/_index/items-2/extendible.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+title = "Extendible"
+weight = 10
+icon = "fas fa-boxes"
++++
+
+With Hugo's resource fallthrough you can create and update fragments and Syna will use the new or updated version of those fragments
```
Comment (stp-ip) [inline exampleSite/content/_index/items-2/extendible.md]: Seems like a small thing, more in line with general flexibility.

File: exampleSite/content/_index/items-2/customizble.md (hunk 5 above)
Comment (mpourismaiel) [reply exampleSite/content/_index/items-2/customizble.md]: If resource pipeline is fixed for netlify and other hosts then the feature would be there. #268

File: exampleSite/content/_index/items-2/extendible.md (hunk 4 above)
Comment (mpourismaiel) [reply exampleSite/content/_index/items-2/extendible.md]: Didn't know the difference actually. Thanks.

File: exampleSite/content/_index/items-1/react.md (hunk 6 above)
Comment (mpourismaiel) [reply exampleSite/content/_index/items-1/react.md]: Wanted to have item fragments for each of the features except for fragments. Then we could just link each column in items fragments to the item fragments and for the fragments column, link it to the fragments list.

File: exampleSite/content/_index/items-2/extendible.md (hunk 7 above)
Comment (mpourismaiel) [reply exampleSite/content/_index/items-2/extendible.md]: I was thinking that flexibility in Syna is that you can reorder nav links, change background colors and other parameters for every fragment. But with extensibility I mean that Syna will use any fragment and it's own fragments can be updated which is actually a Hugo feature but for the uninitiated, it's just another attraction.

File: exampleSite/content/_index/items-2/customizble.md (hunk 5 above)
Comment (stp-ip) [reply exampleSite/content/_index/items-2/customizble.md]: True, but currently it's not part of the feature set ;)

File: exampleSite/content/_index/items-1/react.md (hunk 6 above)
Comment (stp-ip) [reply exampleSite/content/_index/items-1/react.md]: Ah ok. Then happy to keep it. As a placeholder for the bootstrap column we can use "open source", if that makes sense.

**Comment by mpourismaiel** [comment]
@stp-ip What is your thoughts on this one? https://github.com/okkur/syna/pull/341#discussion_r214985676

File: exampleSite/content/_index/items-1/react.md (hunk 6 above)
Comment (stp-ip) [reply exampleSite/content/_index/items-1/react.md]: Will do another review after v0.13.0 is released, but I think it should be fine with pipelines merged.

**Pull request merged**